website/docs/guides/dbt-python-snowpark.md

```diff
@@ -1720,7 +1720,7 @@ We test data models for mainly two reasons:
 - Ensure that our source data is clean on ingestion before we start data modeling/transformation (aka avoid garbage in, garbage out problem).
 - Make sure we don’t introduce bugs in the transformation code we wrote (stop ourselves from creating bad joins/fanouts).
 
-Testing in dbt comes in two flavors: [generic](/docs/build/tests#generic-tests) and [singular](/docs/build/tests#singular-tests).
+Testing in dbt comes in two flavors: [generic](/docs/build/data-tests#generic-tests) and [singular](/docs/build/data-tests#singular-tests).
 
 You define them in a test block (similar to a macro) and once defined, you can reference them by name in your `.yml` files (applying them to models, columns, sources, snapshots, and seeds).
```

Comment:
We might want to add more explicit "data" test callouts to this page - we will not be supporting unit testing python models in our initial release.